admin_console/views/scripts/partner/list.phtml

```diff
@@ -84,10 +84,17 @@
 								<?php if ($partner->status != Kaltura_Client_Enum_PartnerStatus::BLOCKED): ?>
 									<?php if ($partner->partnerGroupType == Kaltura_Client_Enum_PartnerGroupType::TEMPLATE && $partner->partnerParentId== 0): ?>
 										<?php if (Infra_AclHelper::isAllowed('partner','manage-template-partners')): ?>
-                                            <option value="manageNew" ><?php echo $this->translate('Manage - new'); ?></option>
+                                            <option value="manageNew" ><?php echo $this->translate('Manage - KMCNG'); ?></option>
 										<?php endif; ?>
 									<?php elseif (Infra_AclHelper::isAllowed('partner','kmc-redirect')) :?>
-                                        <option value="manageNew" ><?php echo $this->translate('Manage - new'); ?></option>
+                                        <option value="manageNew" ><?php echo $this->translate('Manage - KMCNG'); ?></option>
+									<?php endif;?>
+								<?php endif; ?>
+								
+        
+								<?php if ($partner->status != Kaltura_Client_Enum_PartnerStatus::BLOCKED): ?>
+									<?php if (Infra_AclHelper::isAllowed('partner','kmc-redirect')) :?>
```

Comment:
why its not ep redirect?

Reply:
there is no such privilege in admin.ini and i am not sure we need to add one